change/@ni-nimble-components-aca919ed-c567-4f7b-8c65-7310408a45b2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Use anchored region in Select and Combobox",
+  "packageName": "@ni/nimble-components",
+  "email": "7282195+m-akinc@users.noreply.github.com",
```

Comment:
Could we add a Chromatic test that exercises this case? We have tests that display combobox in an open state; could we add a similar configuration but inside a div?

Reply:
Added